src/test/scala/chiselTests/RangeSpec.scala

```diff
@@ -8,10 +8,64 @@ import org.scalatest.{Matchers, FreeSpec}
 class RangeSpec extends FreeSpec with Matchers {
   "Ranges can be specified for UInt, SInt, and FixedPoint" - {
     "to specify a UInt" in {
-      val x = UInt(range"[0, 7)")
+      val x = UInt(range"[0, 8)")
```

Comment:
I did check it, but for some reason put it in the test that was checking for illegal behavior. I moved it